workshop/SCHEDULE.md

```diff
@@ -5,15 +5,85 @@ nav_title: Schedule
 
 <!--See an example from a past remote workshop here: https://github.com/AlexsLemonade/2023-may-training/blob/main/workshop/SCHEDULE.md -->
 <!--See an example from a past in-person workshop here: https://github.com/AlexsLemonade/2023-june-training/blob/main/workshop/SCHEDULE.md -->
+Below is a preliminary schedule for the 2024 August Introduction to Bulk RNA-Seq and Reproducible Research Practices Workshop.
+This schedule is subject to minor changes and will be updated with additional materials before the workshop.
 
+The workshop will takes place from August 19-22, 2024 at the University of Minnesota.
+We will meet each day at 717 Delaware Street SE,
+Minneapolis, MN 55414 in Room 105.
 
-| Time        | Topic                                          |
+*Note: All times are CDT.*
+
+| Time        | Topic                                          | 
+|-------------|------------------------------------------------|
+| **Day 1**   | **2024-08-19 <br> Intro to R and the Tidyverse <br> Intro to Reproducible Research Practices**|
+| 8:30 AM     | Arrive and check-in <br> Breakfast and coffee available | 
+| 9:00 AM    | Welcome, introductions and getting started |
+| 10:00 AM     | [Introduction to R and RStudio](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) | 
```

Comment:
You can actually remove all of these links. We now copy these notebooks into the `year-month-training` repos instead of using links to `training-modules`, so the links are different altogether now. But, keep the topics, and we'll add in the links after notebooks.

So, do this all the way down the line:

```suggestion
| 10:00 AM     | Introduction to R and RStudio | 
```